tests/jupyterhub_test.py

```diff
@@ -16,11 +16,33 @@ def setup_module(module):
 
 
 def test_check_auth_jupyterhub(fc):
+    import sirepo.srdb
+    from pykern import pkio
+    from pykern.pkunit import pkeq
+
+    def _get_num_jupyterhub_user_dirs():
+        return len(pkio.sorted_glob(sirepo.srdb.root().join("jupyterhub/user/**"))) - 1
+
+    def _jupyter_user_dir_exists(uid):
+        for x in [
+            os.path.basename(d)
+            for d in pkio.sorted_glob(sirepo.srdb.root().join("jupyterhub/user/**"))
+        ]:
+            if uid.lower() in x:
+                return True
+        return False
+
+    fc.sr_logout()
+    n = _get_num_jupyterhub_user_dirs()
+    fc.sr_get("checkAuthJupyterHub")
```

Comment:
I think we should check that we get back a login redirect.

Reply:
[b92ef48](https://github.com/radiasoft/sirepo/pull/6921/commits/b92ef4813f20692833d3b820375d74e338453758)